src/containers/add-documents/AddDocuments.tsx

```diff
@@ -29,7 +30,8 @@ const AddDocuments: FC<AddDocumentsProps> = ({
   buttonText,
   variant,
   sx = {},
-  icon
+  icon,
+  keepPreviousFiles = true
```

Comment:
It is better to set the props in that way so the initial value will be false. Then you can be sure it will not affect old code.

like removePreviousFiles = false